content/docs/use-cases/data-registries.md

```diff
@@ -8,36 +8,34 @@ commands like `dvc import` and `dvc get`. This means that your projects can
 depend on data from other DVC repositories, **similar to a package management
 systems for data science**.
 
-![](/img/data-registry.png) _Data and models as code_
```

Comment:
please, let's not touch data registry in this PR

Reply:
OK please ignore these changes on Data Registries here for now. It's a little harder to roll back right now for me (git stash mess) but it will be reverted ASAP.

Reply:
OK I finally rolled back the changes to Data Regs. Resolving this one